secure/key-manager/src/main.rs

```diff
@@ -75,12 +72,28 @@ fn create_and_execute_key_manager(
     .execute()
 }
 
-fn create_libra_interface(json_rpc_address: SocketAddr) -> JsonRpcLibraInterface {
-    let json_rpc_url = format!("https://{}", json_rpc_address.to_string());
+fn create_libra_interface(json_rpc_url: String) -> JsonRpcLibraInterface {
     info!(
         "Creating a libra interface that talks to the JSON RPC endpoint at: {:?}.",
         json_rpc_url.clone()
     );
-    let json_rpc_client = JsonRpcClient::new(json_rpc_url);
-    JsonRpcLibraInterface::new(json_rpc_client)
+    JsonRpcLibraInterface::new(json_rpc_url)
+}
+
+fn create_secure_storage(secure_backend: SecureBackend) -> Box<dyn Storage> {
+    match secure_backend.clone() {
+        SecureBackend::InMemoryStorage => info!("Using an in memory storage backend."),
```

Comment:
@davidiw, FYI: I added these info!() log calls back to the main function. For some reason I think they were removed by an old PR but I actually find them pretty useful for debugging and when starting up they key manager.